app/selfserve/test/Olcs/src/Mvc/CookieBannerListenerTest.php

```diff
@@ -65,7 +65,7 @@ public function setUp(): void
     public function testAttach(): void
```

Comment:
Mixed feelings on this file :)

Replacing shouldReceive with expects is a good move, as we're making sure things are actually being called, shouldReceive without a once() or times() doesn't give us that...

Less keen on the use of allows() - just about any test can pass with allows(), because it's just saying something may or may not happen. Am understanding of the fact using expects(), specifying the arguments and what's returned will take longer, but it's worth it, because we'll specifically know what we think is being called/returned is actually what's happening.